tests/test_other.py

```diff
@@ -11157,8 +11170,9 @@ def test_unistd_open(self):
     self.do_run_in_out_file_test('wasmfs/wasmfs_open.c')
 
   @also_with_wasmfs
-  def test_unistd_stat(self):
-    self.do_runf(test_file('wasmfs/wasmfs_stat.c'))
+  def test_unistd_fstat(self):
+    self.set_setting('WASMFS')
+    self.do_run_in_out_file_test('wasmfs/wasmfs_fstat.c')
```

Comment:
Looks like this chunk was added by mistake?

Reply:
Oops, sorry I'm not sure what's going on, seems that someone else's changes are regarded part of this PR?